src/normalize/text.ts

```diff
@@ -40,25 +40,21 @@ export default function normalizeText(editor: Editor, entry: NodeEntry): boolean
     }
 
     if (Title.isTitle(node)) {
+        // Title shouldn't be followed by another title.
+        const [next, nextPath] = Editor.next(editor, { at: path }) ?? []
+        if (Title.isTitle(next)) {
+            Transforms.mergeNodes(editor, { at: nextPath })
+            return true
+        }
+
```

Comment:
This should be checked for the title which is in incorrect position, otherwise titles moved to incorrect position will never be normalized.

Reply:
Do you mean: ?
```js
const [prev, prevPath] = Editor.next(editor, { at: path }) ?? []
        if (Title.isTitle(prev)) {
            Transforms.mergeNodes(editor, { at: prevPath })
            return true
  }
```
It is for our test cases with multiple titles

Reply:
actually, it is working, I could swear that it was failing :/